src/main/java/duke/Commands/AddInput.java

```diff
@@ -0,0 +1,106 @@
+package duke.Commands;
+
+import duke.Exceptions.*;
+import duke.Storage.Storage;
+import duke.TaskList.TaskList;
+import duke.Tasks.Task;
+import duke.Parser.Parser;
+import duke.Ui.Ui;
+
+import java.util.Scanner;
+
+/**
+ * The AddInput method adds to a given TaskList based on the command given
+ * saves the list of Tasks in the TaskList into a txt file
+ */
```

Comment:
Perhaps, the class header should be more of what to expect of the class rather than a description of one of its methods